License-header/License-header.txt

```diff
@@ -1,20 +1,25 @@
 /*
  * MORTAR - MOlecule fRagmenTAtion fRamework
- * Copyright (C) $today.year  Felix Baensch, Jonas Schaub (felix.baensch@w-hs.de, jonas.schaub@uni-jena.de)
+ * Copyright (C) 2024  Felix Baensch, Jonas Schaub (felix.baensch@w-hs.de, jonas.schaub@uni-jena.de)
```

Comment:
Why this?

Reply:
Because this placeholder didn't really seem to work.